bindings/python/google_benchmark/benchmark.cc

```diff
@@ -130,18 +130,25 @@ NB_MODULE(_benchmark, m) {
       .value("kAvgIterationsRate", Counter::Flags::kAvgIterationsRate)
       .value("kInvert", Counter::Flags::kInvert)
       .export_values()
-      .def(nb::self | nb::self);
+      .def("__or__", [](Counter::Flags a, Counter::Flags b) {
+        return static_cast<int>(a) | static_cast<int>(b);
+      });
 
   nb::enum_<Counter::OneK>(py_counter, "OneK")
       .value("kIs1000", Counter::OneK::kIs1000)
       .value("kIs1024", Counter::OneK::kIs1024)
       .export_values();
 
   py_counter
-      .def(nb::init<double, Counter::Flags, Counter::OneK>(),
```

Comment:
In your experience, is there any situation left at this point where you would prefer `nb::init<>` over `.def("__init__", ...)`?

Reply:
Well, I'd prefer it wherever it works. But... that's not the case here.